src/pages/home/components/feature-information/index.js

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from 'react'
+import Feature from './feature'
+import { spaces, breakpoints, colors, textStyles } from '../../../../styles'
+import styled from 'styled-components'
+import Carousel from 'nuka-carousel'
+import Media from 'react-media'
+
+const Headline = styled.h2`
+  margin-bottom: ${spaces.relax}px;
+  margin-top: ${spaces.cozy * 6}px;
+  text-align: center;
+
+  ${textStyles.h2Bold}
+
+  @media screen and (min-width: ${breakpoints.desktop}px) {
+    text-align: center;
+    ${textStyles.h1Bold}
+  }
+`
+
+class FeatureInformation extends Component {
+  render() {
+    return (
+      <div>
+        <Media query={`(max-width: ${breakpoints.desktop}px)`}>
+          {matches =>
+            matches ? (
+              <>
+                <Headline>Everything You Need In One Place</Headline>
+                <Carousel
+                  renderCenterLeftControls={({ previousSlide }) => (
+                    <button onClick={previousSlide}>
+                      <img
+                        alt='leftArrow'
+                        src='https://res.cloudinary.com/apomarn/image/upload/v1559083267/WeDo/left-arrow.png'
+                      />
+                    </button>
+                  )}
+                  renderCenterRightControls={({ nextSlide }) => (
+                    <button onClick={nextSlide}>
+                      <img
+                        alt='rightArrow'
+                        src='https://res.cloudinary.com/apomarn/image/upload/v1559083266/WeDo/right-arrow.png'
+                      />
+                    </button>
+                  )}
+                >
+                  <Feature
```

Comment:
don't repeat yourself use a `map` or plain loop to create the feature components.